packages/xlayers/src/app/editor/viewer/lib/sketch-container.component.ts

```diff
@@ -78,10 +78,11 @@ export class SketchContainerComponent implements OnInit {
     try {
       const data = await this.service.process(file);
       this.store.dispatch(new CurrentFile(data));
-    } catch (e) {
+    } catch (error) {
+      console.error(error);
```

Comment:
I would remove this unless there is a specific reason for it?

Reply:
The reason is that if a runtime error occurs, we wont see it. If we ask sone user for an error message in their console, they wont see anything.